src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -141,6 +147,10 @@ public bool TryProcessCommand(IExecutionContext context, string input, Container
 
             return true;
         }
+
+        internal static bool EnhancedAnnotationsEnabled(IExecutionContext context) {
```

Comment:
`private bool`?

Reply:
or do we need a method for a single line of code? 😆

Reply:
lol. My only issue was the `?? false` which could be changed in one place and not the other.

Reply:
because this method is called in 2 places and if there would be 2 instances of `?? false`.  There is potential for them not to match if someone were to change it to `?? true` on one of them